.github/workflows/pull-request.yml

```diff
@@ -55,6 +71,12 @@ jobs:
           docker-compose -f docker/docker-compose.yml up -d app-for-playwright
           npx playwright install chromium
           npx playwright test
+      - name: Upload test results
+        if: always()
+        uses: actions/upload-artifact@v2
```

Comment:
@myieye Could you add a description how these uploads can be downloaded?

Reply:
@JeanneSon can you link me to the run? The cache only gets written to if the whole job was successful. Perhaps it failed?

Reply:
@JeanneSon do you mean at a description/comment in the code?

You can see an artifact near the bottom of this page: https://github.com/JeanneSon/fork-web-languageforge/actions/runs/3096685403

I just realized that, beacuse playwright doesn't save test-results for successful executions (and there's not artifact to upload/download) we'd never be able to compare a failure to a success. Perhaps that's not so great. 🤔